javascript/javascript3/week3/github/github.js

```diff
@@ -0,0 +1,64 @@
+console.log("AB9");
+// HTML ELEMENTS
+function rdm() {
+  return Math.floor(Math.random() * 20) + 1;
+}
+const fullname = document.querySelector(".fullname-1");
+const fullname2 = document.querySelector(".fullname-2");
+const fullname3 = document.querySelector(".fullname-3");
+const owner = document.querySelector(".owner-1");
+const owner2 = document.querySelector(".owner-2");
+const owner3 = document.querySelector(".owner-3");
+const repo = document.querySelector(".url-1");
+const repo2 = document.querySelector(".url-2");
+const repo3 = document.querySelector(".url-3");
+const avatar = document.querySelector(".img-1");
+const avatar2 = document.querySelector(".img-2");
+const avatar3 = document.querySelector(".img-3");
+
+/// Getting into promises
+function first(data) {
+  avatar.src = data.items[1].owner.avatar_url;
```

Comment:
Declare variables, without declaring it like here, that variable automatically becomes a global variable, so return them if you need value out of the function.